hypervisor/arch/x86/guest/vm.c

```diff
@@ -162,9 +162,9 @@ static struct acrn_vm *get_highest_severity_vm(void)
 /**
  * @pre vm != NULL && vm_config != NULL && vm->vmid < CONFIG_MAX_VM_NUM
  */
-bool is_highest_severity_vm(const struct acrn_vm *vm)
+bool is_highest_severity_vm(const struct acrn_vm *vm, bool runtime)
```

Comment:
somehow,  passing an parameter to next level callee API is not decent to me. I felt we can remove this API, and let the caller side to do the check.. How do you think?

Reply:
I'm OK with both. If you like, I could remove this function and check in the caller side.